calibrate.py

```diff
@@ -265,6 +283,25 @@ def show(position, text):
         # cv2.imshow("right", info_frame)
         cv2.imshow("left + right",info_frame)
         cv2.waitKey(2000)
+    
+    def show_failed_orientation(self):
+        width, height = int(self.width * self.output_scale_factor), int(self.height * self.output_scale_factor)
+        info_frame = np.zeros((height, width, 3), np.uint8)
+        print("py: Capture failed, Swap the camera's ")
+
+        def show(position, text):
+            cv2.putText(info_frame, text, position, cv2.FONT_HERSHEY_TRIPLEX, 0.7, (0, 255, 0))
+
+        show((60, int(height / 2 - 40)), "Calibration failed, Left and ")
+        show((60, int(height /2)), "right camera are swapped!")
+        show((60, int(height / 2 + 40)), "Fix \"swap_left_and_right_cameras\"")
+        show((60, int(height / 2 + 80)), "and start again")
+
+        # cv2.imshow("left", info_frame)
+        # cv2.imshow("right", info_frame)
+        cv2.imshow("left + right",info_frame)
+        cv2.waitKey(5000)
```

Comment:
Maybe it would be better to use `cv2.waitKey(0)` that will just wait for a keypress without a time limit